plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/PluginUtil.kt

```diff
@@ -24,6 +25,24 @@ internal inline fun <reified T : Task> Project.taskHelper(name: String, noinline
     return this.tasks.create(name, T::class.java, configuration)
 }
 
+internal inline fun <reified T : Task> Project.maybeCreateTask(
+    name: String,
+    noinline configuration: T.() -> Unit
+): T {
+    return tasks.maybeCreate(name, T::class.java).apply { configure(closureOf(configuration)) }
+}
+
+// Port from Kotlin DSL
+fun <T> Any.closureOf(action: T.() -> Unit): Closure<Any?> = KotlinClosure1(action, this, this)
```

Comment:
@JLLeitschuh Not sure that I did it right here, but IDE failed to find `closureOf {}`

Reply:
I'm not totally certain why you are choosing to use `maybeCreateTask`..
I don't think that anyone else will be creating these tasks, and if they are, then this plugin should fail loudly.

Reply:
You are right, `maybeCreateTask` is not needed in this case.